source/appModules/soffice.py

```diff
@@ -340,3 +340,30 @@ def event_NVDAObject_init(self, obj):
 			# This is a word processor document.
 			obj.description = None
 			obj.treeInterceptorClass = SymphonyDocument
+
+	def searchStatusBar(self, obj, max_depth=5):

```

Comment:
could all these added functions please get type hints added to the headers?

For example:
```suggestion
	def searchStatusBar(self, obj: NVDAObject, max_depth: int = 5) -> Optional[NVDAObject]:
```